From ee52e81f4409b0fccb0342333fb647b9f6c73429 Mon Sep 17 00:00:00 2001
From: John Lees <lees.john6@gmail.com>
Date: Wed, 18 Dec 2024 15:14:42 +0000
Subject: [PATCH] Fix for conda build on OS X

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ec1969d..253d194 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,6 +22,9 @@ endif()
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fno-fast-math")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-fast-math")
 
+# Fix for conda-forge filesystem::path
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_LIBCPP_DISABLE_AVAILABILITY")
+
 if(CMAKE_BUILD_WITH_FLTO)
   cmake_policy(SET CMP0069 NEW)
   set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
-- 
2.39.3 (Apple Git-146)

